### PR TITLE
Add com.github.taiko2k.avvie

### PR DESCRIPTION
**Pull request by Taiko2k**
I'd like to add my app to Flathub. It's a small image manipulation tool.

### CODE REVIEW

File: com.github.taiko2k.avvie.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   "sdk":"org.gnome.Sdk",
+   "command":"run.sh",
+   "finish-args":[
+      "--share=ipc",
+      "--socket=wayland",
+      "--socket=fallback-x11",
+      "--filesystem=xdg-pictures",
+      "--filesystem=xdg-download",
+      "--talk-name=org.freedesktop.Notifications",
+      "--talk-name=org.freedesktop.FileChooser"
```
Comment (barthalion) [inline com.github.taiko2k.avvie.json]: Do you actually need these two?

**Comment by barthalion** [comment]
bot, build com.github.taiko2k.avvie

File: com.github.taiko2k.avvie.json (hunk 1 above)
Comment (Taiko2k) [reply com.github.taiko2k.avvie.json]: I want to have file **drag and drop** functionality. So I picked those two as a compromise in not exposing the whole host but common places the user might want to load file from. Also I have a **one click save** feature so I want to be able to write immediately to relevant locations.

Comment (Taiko2k) [reply com.github.taiko2k.avvie.json]: I see that talk-FileChooser isn't necessary in order to access the portal. My mistake, I've gone ahead and removed that one.

Comment (barthalion) [reply com.github.taiko2k.avvie.json]: :ok_hand:

**Comment by barthalion** [comment]
bot, build com.github.taiko2k.avvie

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/com.github.taiko2k.avvie


Thanks!

**Pull request closed without merging**